<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Pump_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5 번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6 번 펌프</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
